src/grants/use-cases/invoke-action.use-case.js

```diff
@@ -36,6 +37,11 @@ export const invokeActionUseCase = async ({
       payload,
       json: true,
     });
+  } else {
+    logger.warn("unsupported method " + method);
```

Comment:
This will double-log the error. Throwing boom will cause the error to be logged by a global error handler
```suggestion
```